.opencode/skill/frontend/SKILL.md

```
@@ -0,0 +1,80 @@
+# Frontend Engineering Skill for apiary
+
+## Purpose
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)